broker-daemon/broker-rpc/wallet-service/get-trading-capacities.js

```diff
@@ -0,0 +1,62 @@
+const { currencies } = require('../../config')
+const { Big } = require('../../utils')
+/**
+ * Grabs the total balance and total channel balance from a specified engine
+ *
+ * @param {Array<symbol, engine>} SparkSwap Payment Channel Network Engine
+ * @return {Object} including symbol and capactiies for sending and receiving in active, pending, and inactive channels
+ */
+async function getEngineTradingCapacities ([symbol, engine]) {
+  const [openChannelCapacities, pendingChannelCapacities] = await Promise.all([
+    engine.getOpenChannelCapacities(),
+    engine.getPendingChannelCapacities()
+  ])
+
+  const activeChannelCapacities = openChannelCapacities.active
+  const inactiveChannelCapacities = openChannelCapacities.inactive
+  const divideBy = currencies.find(({ symbol: configSymbol }) => configSymbol === symbol).quantumsPerCommon
+  return {
+    symbol,
+    activeSendCapacity: Big(activeChannelCapacities.localBalance).div(divideBy).toString(),
+    activeReceiveCapacity: Big(activeChannelCapacities.remoteBalance).div(divideBy).toString(),
+    pendingSendCapacity: Big(pendingChannelCapacities.localBalance).div(divideBy).toString(),
+    pendingReceiveCapacity: Big(pendingChannelCapacities.remoteBalance).div(divideBy).toString(),
+    inactiveSendCapacity: Big(inactiveChannelCapacities.localBalance).div(divideBy).toString(),
+    inactiveReceiveCapacity: Big(inactiveChannelCapacities.remoteBalance).div(divideBy).toString()
+  }
+}
+
+/**
+ * Grabs the daemons lnd wallet balance
+ *
+ * @function
+ * @param {GrpcUnaryMethod~request} request - request object
+ * @param {Object} request.params
+ * @param {Map} request.engines
+ * @param {Logger} request.logger
+ * @param {function} responses.GetTradingCapacitiesResponse
+ * @return {GetTradingCapacitiesResponse}
+ */
+async function getTradingCapacities ({ params, logger, engines }, { GetTradingCapacitiesResponse }) {
+  const [ baseSymbol, counterSymbol ] = params.market.split('/')
```

Comment:
do we need to check that this is a valid market?

Reply:
Is there another place that we do this?